docs/Buttons.md

```diff
@@ -298,7 +298,20 @@ const ListActions = () => (
 
 ## Record Buttons
 
-### `<DeleteButton>`
+### `<DeleteWithConfirmButton>`
+
+Delete the selected element. To be used inside a <Toolbar> conponent.
+
+```jsx
+import * as React from 'react';
+import { Toolbar, SimpleForm } from '../form';
+
+const EditToolbar = props => (
+    <Toolbar {...props}>
+        <DeleteWithConfirmButton translateOptions={translateOptions} />
```

Comment:
This isn't explanatory. What is the format of translateOptions? Why would anyone use it?

Reply:
I added some elements to be more explicit